app/presenters/notices/setup/preview.presenter.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+/**
+ * Formats notification data ready for presenting in the preview notification page
+ * @module PreviewPresenter
+ */
+
+const NotifyPreviewRequest = require('../../../requests/notify/notify-preview.request.js')
+const { titleCase } = require('../../base.presenter.js')
+
+/**
+ * Formats notification data ready for presenting in the preview notification page
+ *
+ * @param {object} notification - The data relating to the recipients notification
+ * @param {string} sessionId - The UUID for returns notices session record
+ *
+ * @returns {Promise<object>} The data formatted for the preview template
+ */
+async function go(notification, sessionId) {
+  const { licences, messageRef, messageType, personalisation, templateId } = notification
+
+  return {
+    address: messageType === 'letter' ? _address(personalisation) : null,
+    backLink: `/system/notices/setup/${sessionId}/check`,
+    caption: _caption(licences),
+    contents: await _notifyPreview(personalisation, templateId),
+    messageType,
+    pageTitle: titleCase(messageRef.replace(/_/g, ' '))
+  }
+}
+
+function _address(personalisation) {
+  const addressLines = [
+    personalisation['address_line_1'],
```

Comment:
Just double check this includes he name as the top line. 

The two presenters should already do this.

Reply:
It does